components/footer/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@spectrum-css/footer",
+  "version": "2.0.3",
```

Comment:
I think we should start this at `1.0.0`, but checking with the team to make sure.

Reply:
@snowystinger I confirmed with the team, this should start at 1.0.0